podcast.rss

```diff
@@ -1021,5 +1022,132 @@
     </item>
     <!-- END: chapter 107 - maoon -->
     <!-- ==================================================================================== -->
+    <!-- BEGIN: chapter 106 -Quraysh -->
+ <item>
+  <title>92. സൂറ ഖുറൈഷ്  -പാരായണം</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class #92
```

Comment:
User HTML [converter's](https://wordhtml.com/) compress option to bring the CDATA to a single line, refer to previous chapters.